content/docs/studio/how-it-works.md

```diff
@@ -53,6 +59,8 @@ https://www.youtube.com/watch?v=5xM5az78Lrg
 - You can set up your CI/CD actions (e.g. GitHub Actions) to run model training
   upon the creation of Git commits, tags or pull requests. You can use [CML] in
   your CI/CD actions for continuous machine learning.
+- You can use [DVCLive] in your CI/CD actions to [send live metrics of model
+  training to Iterative Studio][live-metrics-and-plots].
```

Comment:
Hmmm, how are these things that Studio does? (This is under **How Iterative Studio runs ... model actions**.)

Reply:
I've merged this point with the previous one; it should be better now.